src/selectors/candidate_filters.js

```diff
@@ -0,0 +1,13 @@
+const { createSelector } = require('reselect')
+
+const getFieldAnalysis = require('./field_analysis')
+
+const getCandidateFilters = createSelector(
```

Comment:
The configurator isn't matching up with the state because of this selector, which is actually to do with some strange things that I do with the filter selector, which I am documenting and fixing, then this will work better for you.